restapi/src/main/java/io/stargate/web/docsapi/service/filter/FilterOp.java

```diff
@@ -0,0 +1,43 @@
+package io.stargate.web.docsapi.service.filter;
+
+import com.google.common.collect.ImmutableSet;
+import io.stargate.db.datastore.query.WhereCondition;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public enum FilterOp {
```

Comment:
Once merged we can probably consolidate this with io.stargate.web.service.FilterOp